src/main/resources/resources/logisim/strings/gui/gui.properties

```diff
@@ -659,6 +659,8 @@ argGeometryOptionArgName = geometry
 argHelpOption = Displays this argument summary help page.
 argLoadOption = Load image file into RAM (works with -tty only).
 argLoadOptionArgName = file
+argSaveOption = Save RAM to image file (works with -tty only).
```

Comment:
As already mentioned, please follow the standards used. Long switches use double dash `--foo`, one letter switches use single dash `-f`. 

Also when adding new localization strings you must edit all related language files (so all `gui-*.properties`) and at least provide placeholders, like this

    # ==> argSaveOption =

Reply:
AH! Thanks for letting me know the convention/rules. I'll submit at new PR.

Reply:
No need for separate PR. Simply push update to this one.

Reply:
I figured that out and did it. Do I need do anything when I do that to notify anyone? (Sorry you have to train me, but thank you.)

Reply:
I believe everything has been corrected in the latest commit.